bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/sensor-thing-types.xml

```diff
@@ -491,6 +491,41 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+
+	<thing-type id="airqualitysensor">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="deconz"/>
+		</supported-bridge-type-refs>
+		<label>Air quality Sensor</label>
+		<description>A air quality sensor</description>
+		<channels>
+			<channel typeId="airquality" id="airquality"/>
+			<channel typeId="airqualityppb" id="airqualityppb"/>
+			<channel typeId="last_updated" id="last_updated"/>
+		</channels>
+
+		<representation-property>uid</representation-property>
+
+		<config-description-ref uri="thing-type:deconz:sensor"/>
+	</thing-type>
+
+	<channel-type id="airquality">
+		<item-type>String</item-type>
+		<label>Air quality level</label>
+		<description>Current air quality level</description>
+		<category>Airquality</category>
+		<state readOnly="true" pattern="%s"></state>
+	</channel-type>
+
+	<channel-type id="airqualityppb">
+		<item-type>Number</item-type>
```

Comment:
Number:Dimensionless?

Reply:
Item-type was changed to Number:Dimensionless.